.github/workflows/lmi-no-code.yml

```diff
@@ -0,0 +1,268 @@
+name: LMI No-Code tests
+
+on:
+  workflow_dispatch:
+    inputs:
+      djl-version:
+        description: 'The released version of DJL'
+        required: false
+        default: ''
+  schedule:
+    - cron: '0 19 * * *'
```

Comment:
let's remove cron job for now until we have some successful runs

Reply:
removed cron for now